cdc/generation.cc

```diff
@@ -330,7 +330,7 @@ topology_description limit_number_of_streams_if_needed(topology_description&& de
     return topology_description(std::move(entries));
 }
 
```

Comment:
Very good, types that refer to the role of the value are better than types that just describe the structure.